docs/README.md

```diff
@@ -0,0 +1,16 @@
+# PyScript documentation
+
+Welcome to the PyScript documentation directory, where PyScript
+is and related topics are discussed.
```

Comment:
Either delete "is" or try this rewrite:

Welcome to the PyScript documentation directory, where you can find and contribute to discussions around PyScript and related topics.